week-4/C-more-complex-objects/exercise-1.js

```diff
@@ -10,8 +10,22 @@ var house = {
     currentOwner: {
       firstName: "Margaret",
       lastName: "Conway"
+    },
+    newAddress: function(newAddress){

```

Comment:
Method names ideally start on a verb so the name indicates the action. What could be a more meaningful name for this method?

Reply:
@foaly-nr1, maybe I should change the name to newOwnerAddress?